includes/validation/class-amp-validation-manager.php

```diff
@@ -469,6 +469,17 @@ public static function add_admin_bar_menu_items( $wp_admin_bar ) {
 			'href'   => esc_url( is_amp_endpoint() ? $non_amp_url : $amp_url ),
 		];
 
+		// Construct admin bar item to link to AMP version or non-AMP version.
+		$paired_browsing_item = [
+			'parent' => 'amp',
+			'id'     => 'amp-paired-browsing',
+			'title'  => esc_html__( 'Start paired browsing', 'amp' ),
+			'href'   => remove_query_arg( amp_get_slug(), add_query_arg( AMP_Theme_Support::PAIRED_BROWSING_QUERY_VAR, '1' ) ),
+			'meta'   => [
+				'target' => 'amp-paired-browsing',
```

Comment:
We can remove the `target` now. Originally I added a `target` to open paired browsing in a new window because there was no link to exit. But now there is. So I don't think we need it.